dissolve-struct-lib/src/main/scala/ch/ethz/dalab/dissolve/classification/BinarySVMWithDBCFW.scala

```diff
@@ -118,9 +119,15 @@ object BinarySVMWithDBCFW extends DissolveFunctions[Vector[Double], Double] {
       data.map {
         case x: LabeledPoint =>
           new LabeledObject[Vector[Double], Double](x.label,
-            if (solverOptions.sparse)
-              SparseVector(x.features.toArray)
-            else
+            if (solverOptions.sparse){
```

Comment:
the LabeledObject should not be dense anyway, but should support standard sparse vector content, no? (this was already in the previous implementation i guess. but i don't really see why to convert from sparse to dense to sparse here